### PR TITLE
Test commit for ksairameshbios/bios/file1

### PR DESCRIPTION
**Pull request by ksairamesh**
Added information to introduce myself as a part of #62

### CODE REVIEW

File: bios/file1 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Hey there
+
+- I live in San Diego
```
Comment (nonolo) [inline bios/file1]: where in san diego?

Comment (ksairamesh) [inline bios/file1]: Updated it to La Jolla, San Diego

**Comment by gregbaxter** [comment]
Looks Good Krithika!

**Comment by gregbaxter** [comment]
Does this mean I merge or do you?

**Comment by gregbaxter** [comment]
Testing notification -- @nonolo @ksairamesh are you sure this code is for R29?

**Pull request merged**